### PR TITLE
feat(EMS-777-779): Accounts - creation - your details page setup, first/last name field validation

### PR DESCRIPTION
**Pull request by ttbarnes**
This PR adds the first page to the "create application" flow/requirement, "your details" page/form.

## Changes
- Create page, controller and routes for "your details" GET and POST.
- Add all form fields to the "your details" page along with other content.
- Add validation rules logic for the first and last name fields.
- Create new shared validation helper function `emptyFieldValidation`; We can use this for any empty field validation rules.

## Other improvements
- Improve account URL constants by prefixing with `INSURANCE_ROOT`, which simplfies testing and consuming the URLs.
- Rename `CREATE_ACCOUNT` URL variable to just `CREATE`.
- Create a cypress command for completing insurance eligibility and starting the "create account" journey.

### CODE REVIEW

File: src/ui/server/controllers/insurance/account/create/your-details/index.ts [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+ * get
+ * Render the Do you already have an account page
+ * @param {Express.Request} Express request
+ * @param {Express.Response} Express response
+ * @returns {Express.Response.render} Do you already have an account page
+ */
+export const get = (req: Request, res: Response) =>
+  res.render(TEMPLATE, {
+    ...insuranceCorePageVariables({
+      PAGE_CONTENT_STRINGS: PAGES.INSURANCE.ACCOUNT.CREATE.YOUR_DETAILS,
```
Comment (Zainzzkk) [inline src/ui/server/controllers/insurance/account/create/your-details/index.ts:61]: can we dereference please as being used more than once

File: src/ui/templates/insurance/account/create/your-details.njk [hunk 2]
```diff
@@ -0,0 +1,151 @@
...
+          value: submittedValues[FIELDS.EMAIL.ID],
+          errorMessage: validationErrors.errorList[FIELDS.EMAIL.ID] and {
+            text: validationErrors.errorList[FIELDS.EMAIL.ID].text,
+            attributes: {
+              "data-cy": FIELDS.EMAIL.ID + "-error-message"
+            }
+          }
+        }) }}
+
+        {{ govukInput({
```
Comment (Zainzzkk) [inline src/ui/templates/insurance/account/create/your-details.njk:108]: does this field show or hide the password?

Comment (ttbarnes) [reply src/ui/templates/insurance/account/create/your-details.njk:108]: Not yet - this is in another ticket.

ATM, just an empty field that doesn't work or do anything, just visual display.

**Review by Zainzzkk — APPROVED** [review]
LGTM

**Pull request merged**